packages/terra-badge/src/_mixins.scss

```diff
@@ -2,3 +2,34 @@
   background-color: map-get($color-map, background-color);
   color: map-get($color-map, color);
 }
+
+$default-theme: 'consumer' !default;
```

Comment:
Moved this into terra-mixins in this commit: https://github.com/cerner/terra-core/pull/732/commits/38794f6d48344762ac35788da9d04a3b16150798